### PR TITLE
Adds weaver-test paragraph to getting-started

### PR DESCRIPTION
**Pull request by Baccata**
As per @djspiewak's request, this adds weaver to the "testing" section of the getting-started documentation. 

It also replaces hardcoded versions of third-parties with scaladex badges (maybe I should keep the versions to facilitate copy paste though, now that I think about it...).

### CODE REVIEW

**Review by vasilmkd — DISMISSED** [review]

**Comment by vasilmkd** [comment]
Regarding the versions, I kind of prefer having a specific version that is easily copy paste-able, but I understand that's difficult to maintain. I will leave the decision to others. (I also wouldn't mind if this PR just bumps the versions to the latest ones).

**Comment by Baccata** [comment]
I mean, that's a good point. Bumped versions in 9acec2c . I left the badges because the information is valuable imho.

**Review by vasilmkd — DISMISSED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -96,9 +100,40 @@ class ExampleSuite extends CatsEffectSuite {
 }
 ```
 
+### Weaver-test
+
+[![weaver-cats Scala version support](https://index.scala-lang.org/disneystreaming/weaver-test/weaver-cats/latest-by-scala-version.svg)](https://index.scala-lang.org/disneystreaming/weaver-test/weaver-cats)
+
+[Weaver](https://github.com/disneystreaming/weaver-test) is a test-framework build directly on top of cats-effect. It is designed specifically to handle thousands of tests exercising I/O layers (http, database calls) concurrently. Weaver makes heavy use of the concurrency constructs and abstractions provided by cats-effect to safely share resources (clients) across tests and suite, and runs all tests in parallel by default.
```
Comment (armanbilge) [inline docs/getting-started.md]:
```suggestion
[Weaver](https://github.com/disneystreaming/weaver-test) is a test-framework built directly on top of Cats Effect. It is designed specifically to handle thousands of tests exercising I/O layers (http, database calls) concurrently. Weaver makes heavy use of the concurrency constructs and abstractions provided by cats-effect to safely share resources (clients) across tests and suite, and runs all tests in parallel by default.
```

**Review by armanbilge — DISMISSED** [review]
Why aren't I using weaver? 😆

File: docs/getting-started.md (hunk 1 above)
Comment (vasilmkd) [reply docs/getting-started.md]: Also, cats-effect in text should be Cats Effect.

Comment (Baccata) [reply docs/getting-started.md]: 7b93721

**Comment by Baccata** [comment]
> Why aren't I using weaver? 

you tell me 😄

**Review by vasilmkd — APPROVED** [review]

**Pull request merged**